pkgs/applications/networking/aether/default.nix

```diff
@@ -0,0 +1,89 @@
+{ autoPatchelfHook, makeDesktopItem, lib, stdenv, wrapGAppsHook
+, alsa-lib, at-spi2-atk, at-spi2-core, atk, cairo, cups, dbus, expat, fontconfig
+, freetype, gdk-pixbuf, glib, gtk3, libcxx, libdrm, libnotify, libpulseaudio, libuuid
+, libX11, libXScrnSaver, libXcomposite, libXcursor, libXdamage, libXext
+, libXfixes, libXi, libXrandr, libXrender, libXtst, libxcb, libxshmfence
+, mesa, nspr, nss, pango, systemd, libappindicator-gtk3, libdbusmenu
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aether";
+  version = "2.0.0-dev15";
+  binaryName = "AetherP2P";
+  desktopName = "Aether";
+
+  src = fetchurl {
+    url = "https://static.getaether.net/Releases/Aether-2.0.0-dev.15/2011262249.19338c93/linux/Aether-2.0.0-dev.15%2B2011262249.19338c93.tar.gz";
+    sha256 = "1hi8w83zal3ciyzg2m62shkbyh6hj7gwsidg3dn88mhfy68himf7";
+    name = "aether-tarball.tar.gz";
+  };
+
+  dontWrapGApps = true;
+
+  nativeBuildInputs = [
+    alsa-lib
+    autoPatchelfHook
+    cups
+    libdrm
+    libuuid
+    libXdamage
+    libX11
+    libXScrnSaver
+    libXtst
+    libxcb
+    libxshmfence
```

Comment:
When I remove that, I get ```/home/mh/.nix-profile/bin/aetherp2p: error while loading shared libraries: libxkbcommon.so.0: cannot open shared object file: No such file or directory```. So I guess it is necessary. Of course I have no clue what `nativeBuildInputs` are because the Nix build system just confuses me most of the time.

Reply:
`buildInputs` and `nativeBuildInputs` are unnecessarily confusing. I hope to improve this with my [suggestion here](https://github.com/NixOS/nixpkgs/issues/28327#issuecomment-879815573).

My best explanation is:
 - `buildInputs` is for stuff that is "like" your own build and can become part of the output
 - `nativeBuildInputs` is for stuff that runs during the derivation's build

However, the distinction isn't currently enforced, and I didn't know that `nativeBuildInputs` libraries end up treated like `buildInputs` libraries when `strictDeps = false` (the default).

All of this is mostly relevant for cross compilation, which isn't actually relevant here, but code like this could cause confusion in other readers, which is why I'm asking.

And I didn't ask clearly. I guess what needs to happen is, could you move the libraries to a `buildInputs` attribute?
Programs that run during the build can stay in `nativeBuildInputs`.

Reply:
Thanks for the explanation. Please re-check

Reply:
lgtm